### PR TITLE
[14.0][IMP] add tag support for brand image

### PR DESCRIPTION
**Pull request by sebastienbeau**
@ivantodorovich to be consistent with product and category

### CODE REVIEW

File: storage_image_product_brand/views/product_brand_image_relation.xml [hunk 1]
```diff
@@ -14,6 +14,12 @@
             <form position="attributes">
                 <attribute name="string">Images</attribute>
             </form>
+            <group name="extra" position="inside">
+                <field
+                    name="tag_id"
+                    context="{'active_model': context.get('active_model')}"
```
Comment (ivantodorovich) [inline storage_image_product_brand/views/product_brand_image_relation.xml:20]: When I saw the same code in the other modules i wondered: is this really necessary?
If `active_model` is already in context, why re-setting it?

**Review by ivantodorovich — APPROVED** [review]

File: storage_image_product_brand/views/product_brand_image_relation.xml (hunk 1 above)
Comment (sebastienbeau) [reply storage_image_product_brand/views/product_brand_image_relation.xml:20]: In fact the field tag_id is doesn't not exist in the abstract model and not in the abstract view.
Maybe we can add it by default ?

Comment (sebastienbeau) [reply storage_image_product_brand/views/product_brand_image_relation.xml:20]: I just realise that I didn't understand you question.
Yes it's needed, indeed odoo do not propagate the context by default for the many2one during the quick create, so we need to pass it explicitly. Just tested right now

Comment (ivantodorovich) [reply storage_image_product_brand/views/product_brand_image_relation.xml:20]: Good to know 👍 thanks!

Comment (simahawk) [reply storage_image_product_brand/views/product_brand_image_relation.xml:20]: good to know, indeed :smile: :pray:

**Review by simahawk — APPROVED** [review]

**Pull request merged**